README.md

```diff
@@ -20,7 +20,13 @@
 - Email: ryo.asaka@g.pascalia.co.jp
 - Hobbies: Sports
 
+# Nobuyuki Ariga
+- DoB: September14 , 1997
+- Email: nobuyuki.ariga@g.pascalia.co.jp
+- Hobbies: Cycling
+
```

Comment:
Because your profile is created after  the person `Quisquam Est`, so that it should be stayed after the `Quisquam Est`. 
Please move your profile to after the last person (The name is Quisquam Est).